src/webui/www/private/scripts/prop-general.js

```diff
@@ -44,32 +44,32 @@ window.qBittorrent.PropGeneral = (function() {
     $("progress").appendChild(piecesBar);
 
     const clearData = function() {
-        $("time_elapsed").set("html", "");
-        $("eta").set("html", "");
-        $("nb_connections").set("html", "");
-        $("total_downloaded").set("html", "");
-        $("total_uploaded").set("html", "");
-        $("dl_speed").set("html", "");
-        $("up_speed").set("html", "");
-        $("dl_limit").set("html", "");
-        $("up_limit").set("html", "");
-        $("total_wasted").set("html", "");
-        $("seeds").set("html", "");
-        $("peers").set("html", "");
-        $("share_ratio").set("html", "");
-        $("popularity").set("html", "");
-        $("reannounce").set("html", "");
-        $("last_seen").set("html", "");
-        $("total_size").set("html", "");
-        $("pieces").set("html", "");
-        $("created_by").set("html", "");
-        $("addition_date").set("html", "");
-        $("completion_date").set("html", "");
-        $("creation_date").set("html", "");
-        $("torrent_hash_v1").set("html", "");
-        $("torrent_hash_v2").set("html", "");
-        $("save_path").set("html", "");
-        $("comment").set("html", "");
+        $("time_elapsed").textContent = "";
```

Comment:
`innerHTML` or `outerHTML` would probably be more equivalent than `textContent`. I'm not sure if it matters since the values are all an empty string. Could only setting the `textContent` leave behind other HTML attributes?

Reply:
Setting `innerHTML` and `textContent` to an empty string lead to equivalent result of wiping element's content.

Setting `outerHTML` would wipe the element itself too, which is not what we want here obviously.